lib/presentation/widget/myspace/my_space_viewmodel.dart

```diff
@@ -513,6 +522,7 @@ class MySpaceViewModel extends BaseViewModel {
     return (Store<AppState> store) async {
       ContextMenuBuilder(context)
           .addHeader(SimpleBottomSheetHeaderBuilder(Key('file_picker_bottom_sheet_header_builder'))
+              .addTransformPadding(Matrix4.translationValues(0, 0, 0.0))
```

Comment:
With value is 0, we don't need it

Reply:
If not provided SimpleBottomSheetHeaderBuilder will give a default value and the header will not be centred